test/dotnet-build3.Tests/GivenDotnetBuild3BuildsCsproj.cs

```diff
@@ -28,10 +28,11 @@ public void It_builds_a_runnable_output()
                 .Should()
                 .Pass();
 
-            var outputDll = Path.Combine(testProjectDirectory, "bin", "Debug", "netcoreapp1.0", $"{testAppName}.dll");
+            var configuration = Environment.GetEnvironmentVariable("CONFIGURATION") ?? "Debug";
+            var outputDll = Path.Combine(testProjectDirectory, "bin", configuration, "netcoreapp1.0", $"{testAppName}.dll");
             var outputRunCommand = new TestCommand("dotnet");
 
-            outputRunCommand.ExecuteWithCapturedOutput(outputDll)
+            outputRunCommand.ExecuteWithCapturedOutput("exec " + outputDll)
```

Comment:
Why is 'exec' needed here?  You should just be able to call `dotnet testapp.dll` right?